4-IMAGES/client/index.html

```diff
@@ -10,16 +11,60 @@
         }
     </style>
 </head>
+
 <body>
-    <img src="http://localhost:8000/image1" alt=''/>
-    <img src="http://localhost:8000/image2" alt=''/>
-    <img src="http://localhost:8000/image3" alt=''/>
-    <img src="http://localhost:8000/image4" alt=''/>
-    <img src="http://localhost:8000/image5" alt=''/>
-    <img src="http://localhost:8000/image6" alt=''/>
-    <img src="http://localhost:8000/image7" alt=''/>
-    <img src="http://localhost:8000/image8" alt=''/>
-    <img src="http://localhost:8000/image9" alt=''/>
-    <img src="http://localhost:8000/image10" alt=''/>
+    <div id="images"></div>
+
+    <script>
+        const imageNames = ['image1', 'image2', 'image3', 'image4', 'image5', 'image6', 'image7', 'image8', 'image9', 'image10'];
+        const resolutions = [320, 768, 1024, 1920];
+        const formats = ['webp', 'jpeg'];
+
+        const imagesDiv = document.getElementById('images');
+
+        imageNames.forEach(name => {
+            const picture = document.createElement('picture');
+
+            resolutions.forEach((res, index) => {
+                formats.forEach(format => {
```

Comment:
Sliced images were included to the page via srcset and media resolutions. Browsers, which support WebP - load images in this format, if browser does not support WebP - images in jpeg format are loaded.